setup.cfg

```diff
@@ -38,6 +38,7 @@ where = src
 [tool:pytest]
 python_files = *_test.py test_*.py *_example.py example_*.py
 addopts = --durations=20 --maxfail=5
+pythonpath = src
```

Comment:
This makes pytest happy on my machine (in shell and in VSCode), otherwise I have to `cd` to `src`, and invoke `pytest ../tests`.

Reply:
I think this should not be needed anymore, if you rebase on the current `main`, you'll see that the `src` folder has been remove, and we have a toplevel `xdsl` folder now!

Reply:
Done